### PR TITLE
Fixed the issue for 'api/users' endpoint when no id is given

### PR DESCRIPTION
**Pull request by hovtoros**
Fixed the issue for 'api/users' endpoint when no id is given, API returns the current user information.

### CODE REVIEW

File: src/api/controllers/users_controller.py [hunk 1]
```diff
@@ -36,19 +36,31 @@ class Users(Resource):
...
-        return jsonify(success=True, data=user_repository.get_all())
+        return make_response(jsonify(
+            {
+                api_constants.SUCCESS: True,
+                api_constants.DATA: map_to_response_user(user, parent)
+            }), 200)
 
 
-@users_controller.route('/users/<string:user_id>', methods=['GET'], doc={'description': USER_GET_BY_ID_CONTROLLER})
+@users_controller.route('/users/info/<string:user_id>', methods=['GET'], doc={'description': USER_GET_BY_ID_CONTROLLER})
```
Comment (hammerhg) [inline src/api/controllers/users_controller.py:57]: Is the UI updated to handle this endpoint correctly?

File: src/api/controllers/users_controller.py [hunk 2]
```diff
@@ -36,19 +36,31 @@ class Users(Resource):
     @jwt_required
     @cross_origin()
     def get(self):
-        user_repository = UserRepository()
```
Comment (hammerhg) [inline src/api/controllers/users_controller.py:39]: Do we have a test that covers this case "no id is given"?

**Review by hammerhg — CHANGES_REQUESTED** [review]

File: src/api/controllers/users_controller.py (hunk 1 above)
Comment (arebecca) [reply src/api/controllers/users_controller.py:57]: The UI integration will be done after this is merged

Comment (hovtoros) [reply src/api/controllers/users_controller.py]: @hammerhg , thanks for the comment, just resolved the endpoint naming to the correct version.

File: src/api/controllers/users_controller.py (hunk 2 above)
Comment (hovtoros) [reply src/api/controllers/users_controller.py:39]: @hammerhg , indeed no.

Comment (hammerhg) [reply src/api/controllers/users_controller.py:39]: Thanks @hovtoros. Please create one. I am going to merge this now.

**Review by hammerhg — APPROVED** [review]

**Pull request merged**